flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/optimize/program/FlinkRuntimeFilterProgramTest.java

```diff
@@ -202,6 +202,64 @@ public void testBuildSideIsJoinWithoutExchange() throws Exception {
         util.verifyPlan(query);
     }
 
+    @Test
+    public void testBuildSideIsJoinWithTwoAggInputs() throws Exception {
```

Comment:
I think it also makes sense to add a test conver only one agg as input case.

Reply:
The test `testBuildSideIsAggWithoutExchange` has covered this case.